### PR TITLE
Fix bug 22 and some translation

### PR DESCRIPTION
**Pull request by mrcuong84**

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -117,22 +117,22 @@ en:
 
     new:
       sign_up: "Sign Up"
-      phone: "Phone"
-      password_confirmation : "Password Confirmation"
+      phone: "Phone*"
```
Comment (dinhvareal) [inline config/locales/en.yml]: `Phone (*)`

File: config/locales/en.yml [hunk 2]
```diff
@@ -117,22 +117,22 @@ en:
...
 
     edit:
       title: "Update user's info"
       update_info: "Update"
       enter_password: "Enter password for confirmation"
 
-    name: "Name"
-    address: "Address"
-    password: "Password"
+    name: "Name*"
```
Comment (dinhvareal) [inline config/locales/en.yml]: `Name (*)`

File: config/locales/en.yml [hunk 3]
```diff
@@ -117,22 +117,22 @@ en:
...
+    address: "Address*"
```
Comment (dinhvareal) [inline config/locales/en.yml]: `Address (*)`

File: config/locales/en.yml [hunk 4]
```diff
@@ -117,22 +117,22 @@ en:
...
+    password: "Password*"
```
Comment (dinhvareal) [inline config/locales/en.yml]: `Password (*)`

File: config/locales/en.yml [hunk 5]
```diff
@@ -117,22 +117,22 @@ en:
 
     new:
       sign_up: "Sign Up"
-      phone: "Phone"
-      password_confirmation : "Password Confirmation"
+      phone: "Phone*"
+      password_confirmation : "Password Confirmation*"
```
Comment (dinhvareal) [inline config/locales/en.yml]: Password Confirmation (*)

File: config/locales/en.yml (hunk 1 above)
Comment (dinhvareal) [reply config/locales/en.yml]: vi.yml ?

**Review by banguyenvareal — APPROVED** [review]

**Review by dinhvareal — APPROVED** [review]

**Review by banguyenvareal — APPROVED** [review]

**Review by hieunguyentrung — APPROVED** [review]

**Pull request merged**